app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadViewModel.kt

```diff
@@ -41,6 +41,8 @@ class ThreadViewModel(application: Application) : AndroidViewModel(application)
 
     private val localSettings by lazy { LocalSettings.getInstance(application) }
 
+    fun getThread(threadUid: String) = ThreadController.getThread(threadUid)
```

Comment:
This is a Realm query, and it will be executed on the main thread. We don't want to do it on this thread.

Reply:
No the case of a get it can pass, it is the case of a transaction or a long query.

Reply:
Ok.